README.md

```diff
@@ -417,21 +419,25 @@ The Galaxy application can be horizontally scaled for the web, job, or workflow
 by setting the desired values of the `webHandlers.replicaCount`,
 `jobHandlers.replicaCount`, and `workflowHandlers.replicaCount` configuration options.
 
-## Galaxy versions
+## Cron jobs
+
+Two Cron jobs are defined to clean up Galaxy's database and the `tmp` directory.  By default these
+jobs run at 02:05 (the database maintenance script) and 02:15 (`tmp` directyory cleanup). Users can 
+change the times the cron jobs are run by changing the `schedule` field in the `values.yaml` file:
 
-Some changes introduced in the chart sometimes rely on changes in the Galaxy
-container image, especially in relation to the Kubernetes runner. This table
-keeps track of recommended Chart versions for particular Galaxy versions as
-breaking changes are introduced. Otherwise, the Galaxy image and chart should be
-independently upgrade-able. In other words, upgrading the Galaxy image from
-`21.05` to `21.09` should be a matter of `helm upgrade my-galaxy cloudve/galaxy
---reuse-values --set image.tag=21.09`.
+```yaml
+cronJobs:
+  maintenance:
+    schedule: "30 6 * * *" # Execute the cron job at 6:30 UTC
+```
 
+To disable a cron job after Galaxy has been deployed simply set the schedule to a date that
+can never occur such as midnight on Februrary 30th:
```

Comment:
Better to add an enabled flag as we have for everything else?

Reply:
Reading more about cron jobs,  maybe I should be using `.spec.suspend` here.  I'll play around with that and see if I get the desired behavior.

Reply:
Good idea to investigate suspend. Maybe we should add both.

Reply:
I think that adding an enabled flag would still be useful. Users who want to run the cronjob manually can still use the trick you suggested. Just that it feels weird to have to think of an invalid date for the simplest case of not wanting the cron job.